overlord/snapstate/snapmgr.go

```diff
@@ -556,15 +549,14 @@ func (m *SnapManager) Ensure() error {
 }
 
 func (m *SnapManager) KnownTaskKinds() []string {
-	return m.runner.KnownTaskKinds()
+	// XXX go away
+	return nil
 }
 
 // Wait implements StateManager.Wait.
 func (m *SnapManager) Wait() {
-	m.runner.Wait()
```

Comment:
AIUI Wait()/Stop() will all become empty for the managers we currently have. What is the plan, will we remove them in a followup?

Reply:
we still call them from the StateEngine on the TaskRunner itself though, we could make them optional I suppose like some parts of the Interface interfaces work